tests/components/snapcast/const.py

```diff
@@ -0,0 +1,87 @@
+"""Constants for Snapcast tests."""
+
+TEST_CLIENT_ENTITY_ID = "media_player.test_client_snapcast_client"
+TEST_GROUP_ENTITY_ID = "media_player.test_group_snapcast_group"
+
+TEST_STATE = {
```

Comment:
I think we could move this to a json fixture and load this when we need it

Reply:
Done